### PR TITLE
[Team] CAES-1393: Add ability to specify role for the new member

### PR DESCRIPTION
**Pull request by sa715**
Signed-off-by: Sergey Alekseyev <sa@4xxi.com>

### CODE REVIEW

File: packages/components/MemberList/components/InviteControl.js [hunk 1]
```diff
@@ -38,12 +45,25 @@ const OPTIONS = Object.values(TEAM_ROLES).map(role => ({
...
+        name="role"
+        value={member.role}
+        options={OPTIONS}
+        className={className}
+        onChange={onChange}
+      />
+    );    
+  }
+  
+  return member.teamIds && member.teamIds.includes(teamId) ? (
```
Comment (cu4xxi) [inline packages/components/MemberList/components/InviteControl.js:66]: you may replace `member.teamIds && member.teamIds.includes(teamId)` with chaining
`member.teamIds?.includes(teamId)`
// nitpick

File: packages/components/MemberList/components/RoleSelector.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { Select } from '@caesar/components/Select';
+
+const StyledSelect = styled(Select)`
+  width: 170px;
+  height: 32px;
+  border: 1px solid ${({ theme }) => theme.color.gallery};
+  margin-left: auto;
+`;
+
+const OPTIONS = Object.values(TEAM_ROLES).map(role => ({
```
Comment (cu4xxi) [inline packages/components/MemberList/components/RoleSelector.js:13]: this is repeatead code.
lets move it into common/constants like a TEAM_ROLES_OPTIONS.
And import here and in `packages/components/MemberList/components/InviteControl.js`.
What do u think?

**Review by cu4xxi — APPROVED** [review]

File: packages/components/MemberList/components/InviteControl.js (hunk 1 above)
Comment (sa715) [reply packages/components/MemberList/components/InviteControl.js:66]: Done

File: packages/components/MemberList/components/RoleSelector.js (hunk 2 above)
Comment (sa715) [reply packages/components/MemberList/components/RoleSelector.js:13]: Done

**Review by beshkenadze — APPROVED** [review]

**Pull request merged**